main.config.js

```diff
@@ -1099,3 +1125,113 @@ export const team = [
     ],
   },
 ];
+
+export const works = [
+      { title: 'Knotel',
+        href: 'project-knotel',
+        imgSrc: 'Knotel.jpg',
+        category: {main: ['Business Digitalization'], additional: ['Real Estate']},
+        descr: 'Headquarters as a Service'},
```

Comment:
@dmytropanontko ,
1. let's replace `descr` with `description`
2. Let's have consistent indentation here like this:

```js
{
...
}, {
...
```